app/controllers/queues_controller.rb

```diff
@@ -0,0 +1,60 @@
+class QueuesController < ApplicationController
+  def spam
+    @request = RescueRequest.find_by(needs_spam_check: nil)
```

Comment:
This will only find one record. Is that what you intended, or do you want multiple?